web/src/main/webapp/resources/css/custom.css

```diff
@@ -0,0 +1,16 @@
+#background {
+	position: fixed;
+	width: 100%;
+	    height: 100%;
+	background: url('blueprint.jpg') no-repeat center top scroll;
+	opacity: 1;
```

Comment:
Creative Commons Zero (CC0) license (from www.pexels.com)
Added comment to CSS as reference.